packages/replayio/src/config.ts

```diff
@@ -5,5 +5,6 @@ export const replayWsServer =
 
 const isCI = !!process.env.CI;
 const isDebugging = !!process.env.DEBUG;
+const isTTY = process.stdout.isTTY;
 
-export const disableAnimatedLog = isCI || isDebugging;
+export const disableAnimatedLog = isCI || isDebugging || isTTY;
```

Comment:
When it would not be disabled now? `isTTY` is our standard mode, right? 🤔

Reply:
🤪 🔫 

Yeah. I did the exact opposite of what I intended to do with that commit.